controllers/clusterresources/redisuser_controller.go

```diff
@@ -132,9 +133,12 @@ func (r *RedisUserReconciler) Reconcile(ctx context.Context, req ctrl.Request) (
 	}
 
 	for clusterID, event := range user.Status.ClustersEvents {
+		userID := fmt.Sprintf(instaclustr.RedisUserIDFmt, clusterID, username)
+
 		if event == models.CreatingEvent {
-			_, err = r.API.CreateRedisUser(user.Spec.ToInstAPI(password, clusterID, username))
-			if err != nil {
+			_, err = r.API.GetRedisUser(userID)
+
+			if err != nil && !errors.Is(err, instaclustr.NotFound) {
 				l.Error(err, "Cannot create a user for the Redis cluster",
```

Comment:
"Cannot create a user..." -> "Cannot get a user..."

Reply:
I thought that it would be strange when customer try to create entity and got message that they cannot get user from the cluster, wdyt?

Reply:
"Cannot create a user for the Redis cluster" --> "Cannot check if the user already exists on the cluster"